manim/mobject/geometry.py

```diff
@@ -392,6 +392,26 @@ def __init__(self, start_point, end_point, **kwargs):
 
 
 class Circle(Arc):
+    """A circle.
+
+    Parameters
+    ----------
+    kwargs : Any
+        additional arguments to be passed to Arc
```

Comment:
For completeness-sake, please fill in the values from the `__init__` down below (e.g.
`         self, color=RED, close_new_points=True, anchors_span_full_range=False, **kwargs`
). It's good to know the types expected by these keyword parameters, if not a short summary of how they change the constructed Mobject.